08 - Fun with HTML5 Canvas/index-CLASS.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>HTML5 Canvas</title>
+</head>
+  <body>
+    <canvas id="draw" width="800" height="800"></canvas>
+    <button class="clearCanvas">Clear Canvas</button>
+    <script>
+        //// classes
+        class DrawingCanvas {
```

Comment:
I love the class. I find it super nice to look at and understand it instantly. I find it a little more descriptive to use [field definitions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes#field_declarations), but these are only supported recently. so good how you did it here.